pipelinewise/cli/pipelinewise.py

```diff
@@ -1072,6 +1072,50 @@ def update_state_file_with_extra_log(line: str) -> str:
             with open(tap.state, 'w', encoding='utf-8') as statefile:
                 statefile.write(state)
 
+    def run_tap_partialsync(self, tap: TapParams, target: TargetParams, transform: TransformParams):
+        """Running the tap for partial sync table"""
+
+        # Build the partial sync executable command
+        command = commands.build_fastsync_partial_command(
```

Comment:
Before running this command, can we have some validation that the given table and column exist and selected? 
We can do that by checking the properties file.

Reply:
Furthermore, we need some validation of the column type, not all column types are suitable to use, I don't think a boolean or string or binary...etc column be useful.

Reply:
- [ ] Add validation for table and column

Reply:
I could not find out how to know the column type from `properties.json` do you know any way for that? 
I also was thinking it can be handled in the sync module when it is going to run the query and if the selected column be an invalid then it would fail and return the error message in that stage. What do you think?

Reply:
Properties file has the json schema of the table (see `schema` key), with the properties (i.e columns) and their type.

Reply:
I reckon we should fail fast, not until the sync starts and query errors.

Reply:
I think `type` is the type of json object and I can see for example for `datetime` fields its type is `string` but `datetime` should be allowed. I was thinking maybe can use `format` field. but I see it is not the thing be included for all objects. 
example: 

```json
    "properties": {
        "date_created": {
            "format": "date-time",
            "inclusion": "available",
            "type": ["null", "string"]
         },
        "date_updated": {
            "format": "date-time",
            "inclusion": "available",
            "type": ["null", "string"]
        },
        "isActive": {
            "inclusion": "available",
            "type": ["null", "boolean"]
        },
        "original_date_created": {
            "inclusion": "available",
            "maxLength": 50,
            "type": ["null", "string"]
         },
        "weight_unit_id": {
            "inclusion": "automatic",
            "maximum": 127,
            "minimum": -128,
            "type": ["null", "integer"]
}
}
```

Reply:
Checking both `type` and `format` (if it exist) should be enough then to make a decision on the validity of the column as a sync key.

Reply:
- [ ] Add checks for type